tools/scripts/run_integration_tests.py

```diff
@@ -40,6 +40,7 @@ def main():
         "aws-cpp-sdk-dynamodb-integration-tests",
         "aws-cpp-sdk-sqs-integration-tests",
         "aws-cpp-sdk-s3-integration-tests",
+        "aws-cpp-sdk-s3-unit-tests",
```

Comment:
im adding to the run integ tests right now. likely this should be a completely seperate step in our CI, and we should have other clients in it. We can create a seperate sep once we have enough client unit tests to justify it.